aten/src/ATen/native/Bincount.cpp

```diff
@@ -0,0 +1,77 @@
+// Returns the frequency of elements of input non-negative integer tensor.
+
+#include "ATen/ATen.h"
+#include "ATen/Dispatch.h"
+
+#include <tuple>
+
+namespace at { namespace native {
+
+#define AT_DISPATCH_BINCOUNT_INT_CASE(scalar, type)                \
+  case scalar: {                                                   \
+    return _bincount_cpu_template<type>(self, weights, minlength); \
+  }
+
+namespace {
+template <typename integral_t>
+Tensor _bincount_cpu_template(
+    const Tensor& self,
+    const Tensor& weights = {},
```

Comment:
This function doesn't need any optional args, because they will be set to concrete value in the one that does the dispatch